.travis.yml

```diff
@@ -0,0 +1,28 @@
+script:
+- echo $APP_KEY $APP_TOKEN
+#Test build
+
+# Docker Build the Image
+- docker build --target test --tag todo-app:test .
+
+# Docker Run to Execute Unit Test Package
+- echo "Running Unit Tests"
+- docker run -e APP_KEY -e APP_TOKEN -e board_id -e list_id_Done -e list_id_Pending -e list_id_ToDo todo-app:test Test/view_model_test.py
+
+# Docker Run to Execute Unit Integration Package
+- echo "Running Integration Tests"
+- docker run -e APP_KEY -e APP_TOKEN -e board_id -e list_id_Done -e list_id_Pending -e list_id_ToDo todo-app:test Test/client_test.py
```

Comment:
The unit and integration tests shouldn't need the real env vars.

And it's better not to pass them in so you never accidentally change that fact.